src/sentry/event_manager.py

```diff
@@ -628,7 +654,7 @@ def _process_existing_aggregate(self, group, event, data):
             extra['culprit'] = data['culprit']
 
         is_regression = False
-        if group.is_resolved() and plugin_is_regression(group, event):
+        if self._is_regression(group, event, release):
```

Comment:
If `release` isn't present, this logic doesn't match the previous behavior — was that intentional?

In the previous version, if `group.is_resolved` returns `True`, but `plugin_is_regression` returns `False`, this would not be identified as a regression, now it is.